sources/web/datalab/polymer/components/item-list/item-list.html

```diff
@@ -29,8 +29,13 @@
       }
       #list-container {
         overflow: auto;
+      }
+      .list-container-no-header--false {
```

Comment:
Well this follows the name of the added attribute: `no-header` that you can add to the `item-list` element to disable the header. I think this makes more sense that defining a `with-header` attribute, which should be the default behavior.